### ISSUE TITLE
Save a PDF

### ISSUE THREAD

**Issue by HowardEdidin** [RickStrahl/MarkdownMonster#352]
```
4/29/2018 7:51:49 PM - Last Resort Handler
Markdown Monster v1.10.14.0
10.0.16299.15.x86fre.rs3_release.170928-1534 - en-US - NET 4.7.1 - 32 bit
en-US - en-US    
---
PresentationFramework
   at MS.Internal.AppModel.ResourcePart.GetStreamCore(FileMode mode, FileAccess access)
   at System.IO.Packaging.PackagePart.GetStream(FileMode mode, FileAccess access)
   at System.IO.Packaging.PackWebResponse.CachedResponse.GetResponseStream()
   at System.IO.Packaging.PackWebResponse.GetResponseStream()
   at System.IO.Packaging.PackWebResponse.get_ContentType()
   at System.Windows.Media.Imaging.BitmapDecoder.SetupDecoderFromUriOrStream(Uri uri, Stream stream, BitmapCacheOption cacheOption, Guid& clsId, Boolean& isOriginalWritable, Stream& uriStream, UnmanagedMemoryStream& unmanagedMemoryStream, SafeFileHandle& safeFilehandle)
   at System.Windows.Media.Imaging.BitmapDecoder.CreateFromUriOrStream(Uri baseUri, Uri uri, Stream stream, BitmapCreateOptions createOptions, BitmapCacheOption cacheOption, RequestCachePolicy uriCachePolicy, Boolean insertInDecoderCache)
   at System.Windows.Media.Imaging.BitmapFrame.CreateFromUriOrStream(Uri baseUri, Uri uri, Stream stream, BitmapCreateOptions createOptions, BitmapCacheOption cacheOption, RequestCachePolicy uriCachePolicy)
   at System.Windows.Media.ImageSourceConverter.ConvertFrom(ITypeDescriptorContext context, CultureInfo culture, Object value)
   at System.Windows.Baml2006.TypeConverterMarkupExtension.ProvideValue(IServiceProvider serviceProvider)
   at MS.Internal.Xaml.Runtime.ClrObjectRuntime.CallProvideValue(MarkupExtension me, IServiceProvider serviceProvider)          
---------------------------

```

**Comment by RickStrahl** [comment]
Can you **please** stop posting things that just come out of hte log without any further context? I see this stuff, but without context these reports are **worthless**.

**Comment by HowardEdidin** [comment]
When using Save as PDF

![image](https://user-images.githubusercontent.com/19954905/39429655-28ce53d8-4c51-11e8-8be3-6e091d9abe1c.png)

**Comment by HowardEdidin** [comment]
![image](https://user-images.githubusercontent.com/19954905/39429717-64a6e942-4c51-11e8-978d-1edc3cd11ebd.png)

**Comment by HowardEdidin** [comment]
Sorry,

I uploaded screen shots in my comments.



Thanks,

Howard

**Comment by RickStrahl** [comment]
This is fixed in 1.10.15+. Dialog was moved and the related updates were not updated.
